.pylintrc

```diff
@@ -50,7 +50,7 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=too-few-public-methods,missing-docstring
+disable=too-few-public-methods,missing-docstring,too-many-return-statements
```

Comment:
if there is a real reason to break codestyle, add inline pylint hint iunstead of modifying pylint config

Reply:
Fixed